src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__gradebook_audit.sql

```diff
@@ -480,6 +470,19 @@ select distinct
         then null
         else iep_status
     end as iep_status,
+    case
+        when
+            audit_flag_name in (
+                'w_percent_graded_completion_by_qt_audit_week_not_100',
+                'f_percent_graded_completion_by_qt_audit_week_not_100',
+                's_percent_graded_completion_by_qt_audit_week_not_100',
+                'w_expected_assign_count_not_met',
+                'f_expected_assign_count_not_met',
+                's_expected_assign_count_not_met'
+            )

```

Comment:
uh... going to take your word for it with these

Reply:
L.O.L.